lib/core/typeinfo.nim

```diff
@@ -372,6 +372,7 @@ proc cmpIgnoreStyle(a, b: cstring): int {.noSideEffect.} =
     else: result = c
   var i = 0
   var j = 0
+  if a[0] != b[0]: return 1
```

Comment:
@xflywind this seems like a bug, shouldn't that be:
`return 1`
=>
`return ord(a[0]) - ord(b[0])`
?

(return value is not binary, it's "ternary" based on sign)

Reply:
Yeah, I was about to revert because of that, but noticed the compilers implementation also does that, so I discarded the branch.